src/logger.ts

```diff
@@ -1,5 +1,10 @@
 import pino from 'pino';
+import type {
+  Logger,
+} from 'pino';
 
-const logger = pino();
+const logger = pino({
+  level: process.env.LOG_LEVEL ?? 'silent',
+});
 
-export { logger };
+export const getLogger = (name: string): Logger => logger.child({ name });
```

Comment:
I am not sure if there will be unintended effects of using `name` rather than another key (e.g. does it break apart or flatten the hierarchy of loggers completely?) but the best part of this change is that this kind of choice is made right here in the single `src/logger.ts` so we can easily change the implementation if new information comes to light.

The logging implementations in the Java world typically have some kind of logger hierarchy so that you can amplify this module and silence that module and so forth in a targeted way. I am not sure how to do that with pino yet but I think I read something in their issues or docs about the `name` being special or related to that capability.

Reply:
That's a great point; `name` is a special option in pino but I don't know if it does anything in the context of children (`ChildOptions` doesn't have a name field for instance).  If our intention is that this is the source file then I think your original `source` field is a fine term.

I'll switch to `source`